website/www/site/content/en/blog/beamquest.md

```diff
@@ -0,0 +1,37 @@
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### **What is the quest?**
+The [Getting Started with Apache Beam quest](https://www.cloudskillsboost.google/quests/310) is a series of four labs that teach and test proficiency in Apache Beam. Upon completion, you're granted a sharable credential badge!
+
+
+### **About the quest**
+
+We’re excited to announce the release of the “Getting Started with Apache Beam” quest, a series of four online labs that venture into different Apache Beam concepts. When you complete all four labs, you’ll earn a [Google Cloud badge](https://www.cloudskillsboost.google/quests/310) that you can share on platforms like LinkedIn. Earning this badge should take less than seven hours total, and signing up for the quest costs $20 (there are often free specials for people who attend Beam events, such as [Meetups](https://www.meetup.com/topics/apache-beam/), [Beam Summit](https://beamsummit.org/), and [Beam College](https://beamcollege.dev/)).
+
+Beam is one of the largest big data open source projects actively in development. Over the past six years, the Apache Beam community has seen tremendous growth in the number  of contributors, committers, and users. If you’re a long time Beam user, you can now earn a badge to show your skills to potential employers. If you’re new to Beam, you can begin your learning journey with this quest. To attempt this quest, you don’t need any prior knowledge of data processing or distributed systems. All you need is elementary knowledge about programming.
+
+Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more, making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.
+
+Given the recent advancements in artificial intelligence, now’s the time to jump into the world of data processing! Get started on your journey [here](https://www.cloudskillsboost.google/quests/310), whether you’re a long time Beam expert or looking to write your first “wordcount” job.
+
+If you have feedback or encounter issues, [reach out](/community/contact-us) or email `support@qwiklabs.com`!
```

Comment:
Do we need this sentence?  Support could be left out, qwiklabs has its on support mechanims.

We can can drop this sentence or change to "If you have any feedback, please [reach out](/community/contact-us)."